Controllers/CommentController.cs

```diff
@@ -0,0 +1,39 @@
+
+
+
+
+
+
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Data;
+using Tabloid.Models;
+using Tabloid.Models.DTOs;
+
+[ApiController]
+[Route("/api/[controller]")]
+public class CommentController : ControllerBase
+{
+
+    private TabloidDbContext _db;
+
+    public CommentController(TabloidDbContext context)
+    {
+        _db = context;
+    }
+
+
+    [HttpPost]
+    [Authorize]
+    public IActionResult AddNewComment(Comment newComment)
```

Comment:
Made a DTO so the `CreationDate` and `Id` properties are not exposed to the client.

Reply:
I'm lost on this part, the Id and CreationDate are never exposed to the front-end as this is posting a new Comment.

Reply:
There aren't fields for those in the form, but if this was being hosted as an API anyone could call this endpoint and fill those fields. You can read more regarding using DTOs to expose data in [this](https://ardalis.com/web-api-dto-considerations/) article under the Create header.